Libplanet/Net/Messages/GetRecentStates.cs

```diff
@@ -12,19 +12,19 @@ internal class GetRecentStates : Message
             BlockLocator baseLocator,
             HashDigest<SHA256> target,
             long offset)
+            : base(LeaveTrail)
         {
             BaseLocator = baseLocator;
             TargetBlockHash = target;
             Offset = offset;
         }
 
         public GetRecentStates(NetMQFrame[] frames)
-            : this(
-                new BlockLocator(frames.Skip(2).Select(f => new HashDigest<SHA256>(f.Buffer))),
-                new HashDigest<SHA256>(frames[1].Buffer),
-                frames[0].ConvertToInt64()
-            )
         {
+            BaseLocator =
```

Comment:
Is this change necessary?

Reply:
Yes, because message generated with `NetMQFrame`s should keep previous breadcrumb trail.

Reply:
Oh, I see. how about reusing `this()` constructor?